packages/aws-cdk-lib/aws-lambda-nodejs/test/bundling.test.ts

```diff
@@ -13,6 +13,7 @@ import * as util from '../lib/util';
 
 const STANDARD_RUNTIME = Runtime.NODEJS_18_X;
 const STANDARD_TARGET = 'node18';
+const LATEST_TARGET = 'node20';
```

Comment:
Can we just change `STANDARD_TARGET` and `STANDARD_RUNTIME` instead of adding `LATEST_TARGET`?